README.md

```diff
@@ -11,22 +11,49 @@
 
 Add [`@PullPartyBot`](https://t.me/PullPartyBot) to the chat to be able to create custom parties and manage them via following commands:
 
+> all commands are not case-sensitive to support RUDE mode \
+> usernames are char sequences of letters, digits and underscores of 5-32 length
+
 + `/create partyName user1 user2...` - create party with mentioned users
 
+    > You can enter users with or without `@` symbol \
+    You can't create empty party \
+    `@admins` party is reserved and already created \
+    `@` prefix will be removed from party name \
+    `@` symbol is not allowed in party name \
+    You can't add single user to one group more than once
+
 + `/update partyName user2 user3...` - update an existing party
 
+    > Inherits `/create` method's rules \
```

Comment:
> Inherits

That supposes some kind of inheritance, which is not quite a concept used here. I would propose "The rules being applied are the same as for `/create` method" or smth like that, u know?